src/app/Shared/URLs.ts

```diff
@@ -9,5 +9,6 @@ export const ZNPaths = {
 export const mockDataURL = "/v1/mockdata";
 export const filterURL = "/v1/filterconfig";
 export const isolateURL = "/v1/isolate";
+export const isolateCountURL = "/v1/isolate/counted/";
```

Comment:
these are constants and should be upper case style

Reply:
Done. Commit: "Change upper case style and remove "event""